how-to/business-logic/how-to-determine-the-user-account.md

```diff
@@ -0,0 +1,56 @@
+---
+title: How to Determine the User Account Which Is Making the Request
+description: An example to perform a check and use the current user account properties
+type: how-to
+page_title: Determine the User Account Which Is Making the Request in {{site.bs-bl}}
+slug: bl-determine-user-account
+position:
+tags: backendservices, user-management, business-logic, cloud-code
+teampulseid: 
+ticketid: 
+pitsid: 
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Business Logic</td>
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Cloud Code for Data, Cloud Functions</td>
+  </tr>
+</table>
+
+## Description
+
+Frequently you need to check the user account properties of the current user making a request to a content type or to a cloud function. 

```

Comment:
* Consider using often instead of frequently.
* You can probably drop "user" from "user account".
* Consider applying the "you may need to" formulation you used in the previous article.
* It might be a good idea to make this "of the current user who is making a request to a content type or a cloud function."